src/librustc_back/target/mod.rs

```diff
@@ -357,6 +358,7 @@ impl Target {
             i386_apple_ios,
             x86_64_apple_ios,
             aarch64_apple_ios,
+            aarch64_linux_android,
```

Comment:
Oh, sure. Thanks for finding this.
But since this commit is already being tested by bors, I'll fix it in my next PR.